ochazuke/webhooks/__init__.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Flask Blueprint for our "webhooks" module.webhooks.
+
+See https://developer.github.com/webhooks/ for what is possible.
+"""
+
+import json
+import logging
+
+from flask import Blueprint
+from flask import request
+
+from ochazuke import app
+
+from .helpers import is_github_hook
+from .helpers import process_issue_event_info
+from .helpers import process_label_event_info
+from .helpers import process_milestone_event_info
+
+webhooks = Blueprint('webhooks', __name__, url_prefix='/webhooks')
+
+
+@webhooks.route('/issues', methods=['POST'])
+def issues_hooklistener():
+    """Listen for the "issues" webhook event.
+
+    By default, we return a 403 HTTP response.
+    """
+    if not is_github_hook(request):
+        return ('Move along, nothing to see here', 401, {'Content-Type':
+                                                         'text/plain'})
+    payload = json.loads(request.data)
```

Comment:
I guess it's probably not very likely, but perhaps we should handle the case where `json.loads()` fails here... like GitHub sent us an empty string or something. Maybe a try/except returning a 403 if it throws?

Reply:
Or maybe `is_github_hook` could check that the response is something we can parse? Once we get past that, we're good to assume we can operate it.